src/main/webapp/WEB-INF/views/car/get_all.jsp

```diff
@@ -0,0 +1,49 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+
```

Comment:
remove the empty line here in all the JSPs

Reply:
Done